src/qt/rpcconsole.cpp

```diff
@@ -466,14 +471,14 @@ RPCConsole::RPCConsole(const PlatformStyle *_platformStyle, QWidget *parent) :
     ui->detailWidget->hide();
     ui->peerHeading->setText(tr("Select a peer to view detailed information."));
 
-    QSettings settings;
     consoleFontSize = settings.value(fontSizeSettingsKey, QFontInfo(QFont()).pointSize()).toInt();
     clear();
 }
 
 RPCConsole::~RPCConsole()
 {
-    GUIUtil::saveWindowGeometry("nRPCConsoleWindow", this);
+    QSettings settings;
+    settings.setValue("RPCConsoleWindowGeometry", saveGeometry());
```

Comment:
Funny fact, in Qt documentation this is done by overriding `QWidget::closeEvent`. Is the geometry available in the destructor in all platforms?

Reply:
The same as above, I would keep `GUIUtil::saveWindowGeometry`, maybe add some logging there too.

Reply:
I disagree, it's pretty excessive to leave one line of code in its own function, and this is pretty inconsequential behavior so I don't think its worth logging, unless others think it is?

Reply:
No, IMO, no need to have utility functions for something that is a one-liner. That just makes code harder to follow.